demo/admin/src/products/future/ManufacturersPage.cometGen.ts

```diff
@@ -0,0 +1,16 @@
+import { type CrudPageConfig } from "@comet/admin-generator";
+import { type GQLManufacturer } from "@src/graphql.generated";
+
+import { ManufacturerForm } from "../ManufacturerForm";
+import { ManufacturersGrid } from "./generated/ManufacturersGrid";
+
+export const ManufacturersPage: CrudPageConfig<GQLManufacturer> = {
+    type: "crudPage",
+    gqlType: "Manufacturer",
```

Comment:
Is the GraphQL type necessary for the page generator?

Reply:
We need some value for texts of the title, toolbar, add-button, and for message-IDs. 
So it doesn't really need to be the GQL-Type but we need something.

We could change it to something like `entityName`, `itemName` 🤔

Reply:
Didn't get that 😅 Let's keep it like this so it's the same for all configs.